pom.xml

```diff
@@ -977,14 +977,6 @@
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>*</artifactId>
           </exclusion>
-        </exclusions>
-      </dependency>
-      <dependency>
-        <groupId>org.apache.hive</groupId>
-        <artifactId>hive-exec</artifactId>
-        <version>${hive.version}</version>
-        <scope>provided</scope>
```

Comment:
Thanks, we already add the hive exec jar in the flink bundle pom, doesn't that work ?